sdk/helper/ldaputil/client_test.go

```diff
@@ -32,6 +32,9 @@ func TestLDAPEscape(t *testing.T) {
 		"test\\hello": "test\\\\hello",
 		"  test  ":    "\\  test \\ ",
 		"":            "",
+		"\\test":      "\\\\test",
+		"test\\":      "test\\\\",
```

Comment:
So, if I'm reading this correctly, previous to your change, passing `test\\` as an input to `EscapeLDAPValue` would cause a panic?

Reply:
Correct, as would passing a single escape character `\\`.